### ISSUE TITLE
Genesis error when trying to use Events

### ISSUE THREAD

**Issue by HollowGraphic** [jeffcampbellmakesgames/unity-coc#5]
Adding `[Event()]` attribute to any component causes Genesis to fail with an error stating "Value cannot be null. Parameter name: element"

**Comment by jeffcampbellmakesgames** [comment]
What is the full type for `[Event()]`? Is this related to **EntitasRedux**? I'm wondering if this is supposed to be an issue on the **Unity COC** library, **Genesis**, or **EntitasRedux**

**Comment by jeffcampbellmakesgames** [comment]
I don't believe **COC** uses **Genesis**, which makes me think this issue might belong on a different repository.

**Comment by HollowGraphic** [comment]
`JCMG.EntitasRedux.EventAttribute` is the full type

**Comment by jeffcampbellmakesgames** [comment]
Cool, thanks @HollowGraphic! I will transfer this issue to that repository and take a look later today. Thanks for reporting!
